app/services/add_conseiller_numerique.rb

```diff
@@ -11,14 +16,16 @@ class Structure
     attr_accessor :name, :address, :external_id
```

Comment:
L'adresse est portée par les lieux désormais, non ?

```suggestion
    attr_accessor :name, :external_id
```